README.md

```diff
@@ -22,14 +22,26 @@ $ export REGISTRY=<SOME_REGISTRY> \
 ```
 2. Update the `deploy/operator.yaml` to pull the Marketplace Operator image you just pushed. You should update the `spec.template.spec.containers[0].image` field with the `$REGISTRY/$NAMESPACE/$REPOSITORY:$TAG` value.
 
-## Deploying the Marketplace Operator
+## Using the Marketplace Operator
 
-### In an OKD Cluster
+### Description
+
+The marketplace operator manages two CRDs: [operatorsource](./deploy/operatorsource.crd.yaml) and [catalogsourceconfig](./deploy/catalogsourceconfig.crd.yaml). When an operatorsource CR is created in the same namespace as where the marketplace operator is running, the marketplace operator will download manifests stored in the external registry specified in this operatorsource CR (for now, please see documentation about using quay's appregistry API). For an example of this operatorsource CR please see the [examples](./deploy/examples/) folder.
+
+The operator will then create a catalogsourceconfig CR which, for the time being, will trigger the marketplace operator to create a configmap CR and catalogsource CR. The catalog-operator and olm-operator managed by [OLM](https://github.com/operator-framework/operator-lifecycle-manager) will then respond to the creation of these CRs and, if a CSV (ClusterServiceVersion) was present in the manifest files, your operator(s) downloaded from the external registry will be manageable by OLM.
+
+### Deploying the Marketplace Operator
+
+#### In an OKD Cluster
 ```bash
-$ oc apply -f deploy
+$ oc apply -f deploy/
 ```
 
-### In a Kubernetes Cluster
+#### In the UI
+
+When deploying OLM, include the contents of the CSV file located in the [deploy](./deploy/) folder in the rh-operators configmap shipped with the latest version of OLM. Follow the remaining instructions for deploying OLM. Finally, navigate to the `package manifest` navigation tab in the `Operators` section, find the marketplace operator, and follow the steps after clicking `Create Subscription` in the OLM UI to deploy the operator.
```

Comment:
This should go in only after we have added the CSV to repo. Moreover once that is in and is part of rh-operators, you only need to mention how to subscribe to it. It should also be called out that this is a OpenShift UI.